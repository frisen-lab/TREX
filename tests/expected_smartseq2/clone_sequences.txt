clone_nr	clone_id
1	--------ATTTATAGGCTCAACATTCAAC
2	-----GCATGGGGGAGGCAATTGGAATTGG
3	---AGGTCGGAGTGTTGTCAGCCAAGAAGA
4	--GTACGTCTGTGGTTGGATTATATTGTAC
5	AAACTTTGGGTTCATGACAATCCTCCACAG
6	AACCTTGCTATGTGTATAGGCGTTCTGTAC
7	AACGTCAATACGCTATCCCAAATTGTTAAC
8	ACCAATAGGAGCCTTAACCGTCGTGGGCCG
9	ACCTGCCCACGCACGGAAAGAGTGTGGGGC
10	AGACGGCACATGTGATTTGAATGGTTGTAC
11	AGCATTAAAGGACGAGGAATTCAGGCCTCC
12	AGGTCGGTCACTACCCGGAAGTGCATATTA
13	AGTCATTCTTACCTCAAGCGGATCTCCGGT
14	AGTGGTAAATGTGACTCTGGGCGCGGGGGT
15	AGTTGTTGCTGAGGAGTCGTTTATCTGGGG
16	ATAGCGCGGTCGGTGGACGATTTCGGATAC
17	ATTAGAATCAGTATGATTATTCATCTATAA
18	ATTCGTGTGCGCAGAATTGAGGGGAAGAG-
19	CAAAATACCGTACCTGCCCTGGGGGCCGAC
20	CCATGGGCAGCAGCAGGGTCTGCGCGGAAG
21	CCGAGTCTGCGGGGTGTCAACAGCTCAAGT
22	CCTGTAATCGGGGGACTTTTAAAGTGGGCG
23	CGCAGTGTAGTAACGTATCGTGGCTTGGTG
24	CGTAATCCTATCGCAATGAGGGTGCTGATC
25	CGTGGGCTCGGAGATGTGTATAAGAGACAG
26	CTACAATGCAGGACACAGACAGATTGCTGT
27	CTAGTAAAAGGAGTATATTCGG--------
28	CTCTTGATAGACCCTGTGGTAATATGTCCG
29	CTGCCACGAATAGCGAGTTTTGGGAGTTAC
30	GAAGGGGGCACGCACTAAATTCCGCAGGAC
31	GAAGTCTTACATTGGGGTAGAC--------
32	GACGTGGGGAGGTGCATTTGTGTATTATGA
33	GAGAATTTGGGTAGGACGCGCCCGAAACTT
34	GAGCGAAAATTGAAATATTGCTTCTTCGGT
35	GAGTGCAAGCCCAATCTTGTGTTTTAGTAC
36	GATGACACGGGGTTGACTAATGATGGTACC
37	GCCTATTCGTATTTAATACGGGCGAATTGG
38	GCGGACTGGATAAGCTGTGCGCAGAACTCG
39	GCGTGTGTAACAAGTGGCTCTACTGCAAGG
40	GCTTGGAACTTTTTGTCCCAGGATGAAACG
41	GGAAGGGGGTTGTAGGGATCGGTAATTCAA
42	GGATGACGGCGTTGGTCCCGG---------
43	GGCACCGGGAGGTATACGCAGGGTCAGGGC
44	GGCCGGCATCTCGTCATCTACTTTTGGCGA
45	GGGGTTGCATATATAGTTACGTGTCTGGTG
46	GTAAAATGAAGTGTGACATGTGCGGCTAGC
47	GTCGGGAAGAACCGAGTGGTGAT-------
48	GTCTAGGCAGACACCATGGGTGCTGGGCAT
49	GTCTCAGTTTATAGTCCTTTGAAGTCGTAC
50	GTGGGAGTTAATGCATTAAAGCCCGCGTAC
51	GTTCTAGGTGCGACAGGAGGGTATCTGGTT
52	GTTGCTAACGCGAATAAACATACGTTAGAG
53	GTTGGTGCTGTAGTAATCCCGAGGGCTTAA
54	TAAAGGTTGGGGACGCAAACATTTTTGGGA
55	TACTAATAATGACATCCGAATGCGTTTTCG
56	TAGGTGTTTAGTGGTGTAAAGGGAACA---
57	TAGTGGGTGAGGGACGAGTCACGGTACCTT
58	TATATGTGCTCCGGCTCTCACTGGGGGTAC
59	TATTGGAGGTTGCTCACCATCTCGCAGCGC
60	TCAGAAGTTACACCGGGTGTTAGTG-----
61	TCAGAGCAGTATTGGTTAAGCGTGTAGGAG
62	TCAGGCTATACGTATAGGGCGAGTGCGTAG
63	TCAGTTCTGAGTGTAATTTCGTTATTTAGC
64	TCATAATGGGGGGCGAGGTATGAC------
65	TCGAAAGTAGAGGTGACCAAGTGAGATGAT
66	TCGATGGTACACTTTACAAGAATTAGGTGC
67	TCGATTATAATTCATGGGAGTGGTTAGTGT
68	TCGCCGGGGTCGTTTACGCCAGGTTTGGGC
69	TCGCGAGGTATATAGCGCAGTTGTAGAT--
70	TCGCGCTTGTATGTTTGGTATTCATATGAC
71	TCGCGTCGAACGTAGCACCTACCAGTGTGC
72	TCGCTACCTGTCTATACGCAGCGCGGGCGG
73	TCGGTCAGAAACCATGGGCTAGGGTTGGAG
74	TCGTCTGGCCCACCGAAGTCAGAGAACGTG
75	TCGTGTAGAGTGTGGTCTATGGGGATCATG
76	TCTAACGGCTCGCCCGGTGACCGTGCATAG
77	TCTCCAGGTTCCGGCGGGGAGAGGTTAGGG
78	TGAAATGGATGCGCAGCCGAAAAGCCGTAC
79	TGATACTGGATATGTTGTGCCAGAGGGCAC
80	TGATCAGGGTATAGGTTACATCAGAGGTAC
81	TGCGATACCGGAGTTTAGCGG---------
82	TGGACATAGCAGACTCCTACGTGGTAAAGG
83	TGGAGAAAGGGTCTTTTCACAATGTGATTG
84	TGTGGACGTACTATTTGGGCTCCGTTACCC
85	TGTGGATTGCACAGACCAGAGC--------
86	TTACCGGATATGTGTGGGTGCTTACTAGCG
87	TTAGCTAAGGTCTCTTAGGTTGGTATCGTG
88	TTGGGGCGTTAGAATGAATACCACTTGGCT
89	TTTCTAGGGGCTTCTTTTATTATGACAGGA
