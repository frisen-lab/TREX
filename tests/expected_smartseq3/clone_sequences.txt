clone_nr	clone_id
1	----------CGAAGGGAAGTGGCACCTTC
2	----------GGCGACGTTATCGGGGAGTA
3	----------TTCGAAAACGAGATTTATCC
4	---------ACAGACAGGGGGCCGTAGGCC
5	---------GCAACCATGGATGAGGCTCGT
6	---------GCATTGTTATAAGGGATGGAT
7	--------ATGGCTCTGCAGGGGCTGTTTG
8	-------ATTGTTTGCAATACCCTAGCGGC
9	-------CAATGGACGGGCGGGGGGCGAAT
10	------AGATCGGCAGTGAGTGACGATCAT
11	-----GGTTACGGCTCGAGGGAGCGAATAC
12	---CAATGGTTGGAGAGGGAATTCCAACAC
13	---CAGATTGCGCAATGAGTTCCGGGGGGA
14	---CAGGGGGCTGTGTAATTATTGTATCGG
15	AAAAAACC---AACCAGCCGATTTGGGTAC
16	AAAGTTTTTTTTGAGCTGGGATGGTTAAAC
17	AAATCGTAACGGAGGGTGCGCGGCCCGAGG
18	AAATGAGCATGAACAACCGAAACAAGGAAC
19	AAATGAGCATGACGAGCTGAACGATGTACC
20	AACAGGAAATGCGAGAGGCTCAAGTTAGAC
21	AACGTGGTGTGTGTCCGGAATGGGTTAGGG
22	AAGAATGTGAATTTGGGGCAAGGGTATCGG
23	AAGAGTTGCAGAATTATAGCATAATGGTTT
24	AAGCTTCCGGATGAGCCCGCAAGGGTGTTA
25	AAGCTTCCGGATGAGCCCGCAAGGGTGTTA
26	AAGCTTCCGGATGAGCCCGCAAGGGTGTTA
27	AAGTAGATATGTATCGGTGTGGTCGCATCA
28	AAGTCTGTTGTTCTATTACCTAAGCACGAG
29	AAGTTTGAGATGGCAAAAGGTTGGGGGTGG
30	AATAGGATGTTCAATGCACGCCGGTGCCAG
31	AATCGCGAGAGAGCGGTAGTGCCAGTGGAC
32	ACAAGGGAATGAAGTTTCGTAGGCTTGGGA
33	ACAGATTGCGCAATGTAGCAAGGGGGGTGG
34	ACCCGCTTGTTGCGAGGAATACATCGTGGG
35	ACGCACTAATTGTGCAACTGTGCGTGTTGT
36	ACGTGTACGGATGCATGGTCGGCTGCCAGG
37	ACTAGTTGTGACCGGTAACTTAAGTACGAC
38	ACTCTATGACCATGGGGAGACAAGTTACAC
39	ACTGTGTATGTTTACTCCACACTAATCTGC
40	AGAAATCCGTATGGTAGCTATTGGTCAAAC
41	AGAAGACTATGATGGGTTTGCCTCGGCGAC
42	AGACAGATTGCGCAATGGGTGAACGGGGAG
43	AGAGTTTCTTCCTTGGGTCGCTGGTTACCG
44	AGATCGTTGGTTCCCGACGCAGGCCGGGCA
45	AGATGGCTTACGAGTGTTCATAGTGTCGAC
46	AGCAGCTACTGATGGGGATGATTTCTGGGG
47	AGCAGCTACTGATGGGGATGATTTCTGGGG
48	AGCGCAATAGTGAAATTGCTCTAGGTTGAC
49	AGGGGTCATACTTTAGTAGGGTAAAGAGCT
50	AGGGTACGCCACGCGTGTCCACCTGTGACC
51	AGGTGGGTGGGGGGGAGGTTTGGGTGGTAC
52	AGTCGAGTCGTTTCATCTCTCGGTGCAGAA
53	AGTCGCGGCAAGGGCTCCGGTGGCAGCAGT
54	AGTGAAAGGGGCGAGTGGGGCAGGAGCGCC
55	AGTGTTGTTGGAGCCGTGTTAAGCTGGTAC
56	AGTTGGTGGATTGGGTCGGTGGTGGCGTAC
57	AGTTTTGGGTTAATTAGAACTAGCGTTTTT
58	ATAAGCCAGGGACTGGAAGGTTAGAGGACA
59	ATCGAGCCCGATTGATGTAGCATCTTGTGC
60	ATCGGAACACCAAGCATGCATAGTACCATC
61	ATGAAAATCCTGTTAAGTTGTACAGGTGAC
62	ATGAAGTGATCACCGGTTGTGTGATGAATA
63	ATGAGGAGGAATGGTTTAGGTCCTGAGGGT
64	ATGGAATTGAGCGCCCGTGTCTCGGAATAG
65	ATGGGCTAGTTCGAGGTAACAGACCTTGTC
66	ATGGGGTAACCGGGATGATTGCGGCGGTAC
67	ATGGGTCCCGGAGTTTGGAGGGGCTCGTAC
68	ATGTTATATGTGGAGTTATATGCTCGGTAC
69	ATGTTATATGTGGAGTTATATGCTCGGTAC
70	ATGTTATATGTGGAGTTATATGCTCGGTAC
71	ATGTTTGAAGGTACAGTACACCGTCGTTGA
72	ATTATGGCAGGTTGTATGAAGTAAGTAAGA
73	ATTGAGTGGATGTAGGATTAGAGTGAGGCG
74	ATTTTTGCGTTAATTTTTATTGTGAACCAA
75	CAAAAGGGGATTTAGTGTGAGGGGGGATGC
76	CAAGACAGTGCTTTTATGTCGTGTCTGTAA
77	CAAGCAAAGGGCGATCGGAACTCTCGGTAC
78	CAATCAAGGAAGTTGTACCCGCATCGGTAC
79	CAGCTTGCGCACTGGGGGCGGGGGGCGAAT
80	CAGGGGCCATCAAAGCCCTCTGGGTTGTAC
81	CAGTGATTGTGGTCCTGTTGGTTGTGGTAC
82	CAGTTCCGAGGGTGCGCGTAGAGAGCCAGG
83	CATCGTATCGGTAGCTGTCTCTTATACACA
84	CATCGTATCGGTAGTGCGGACCGGGATTGA
85	CATGTCACTATTGAATGTGGATCGAGGGGC
86	CATGTTCTAGTGTCACCACTCATTGGAAAC
87	CCAAAGGGCGAGTGGATGCGTGACGGGTAC
88	CCAATTAGCTCGACGCGACTGCGGATTAAA
89	CCGAAGCGCTATGCTGCACTGCAACCCCGC
90	CCGCATCCAGATCATGCAAAGTTATTGTA-
91	CCGGATAGCGCATTATATGAGAATTAGTAA
92	CCGGCGCCGAGAGGGAGGTACTTATTGTAC
93	CCGTTTGATCTGGGGATTGGGTGGCATTAC
94	CCTATGGACTGTGTACTAGGAGAATGGTTG
95	CCTCAGATCCTCGCGCTGTGGTCCACGTAC
96	CCTCATTGCAGAGGAAAGTGCTTGGCCGTA
97	CCTCTGAGTGCCACTCCCAATGGTCGGTAC
98	CCTGTACTTCAAACTGGTTTACCAGAAGTG
99	CGAAGGGATCTGCGGGTTGTGGTTGTGTAC
100	CGACTAGGGTGGGACGGCAGCTCCAGGGGG
101	CGAGGCGTGCGTTTAGCTCATGTTCCTTTG
102	CGATACGATGTGGGGCAGGGTGGATATGTC
103	CGCAATGAAGGGGGCAGGGTGGATATGTAC
104	CGCAATGCGATTGCCGTGTGAATTGCAATG
105	CGCAATGTAGAGGGGGGGAGAGCCGCGTGG
106	CGCATTTCCGTGGTCTAATTTTAGATGTAC
107	CGCATTTCCGTGGTCTAATTTTAGATGTAC
108	CGCTGGGTTGTGGATGAAAAAAGTCAGGGG
109	CGCTTAAAGTGGTGGGGCGCTATGGTTAGG
110	CGGAGGTGTGCCGGGCCGAGGATTCACAGG
111	CGGGCGCCTTGACGGAAGCAATTGCCGTAC
112	CGGGGGAGCTGGTCAGGTTGAACGCGAAGG
113	CGGTCGGGGTGGAACTAACCGTTTGCACGA
114	CGGTTCAGTTGGAAAATATCGTGGAATCAC
115	CGTGTGTGAAGGAACCTCACAATAGCGTGT
116	CGTTGTTTGCGTGGTTCACGTAATGAAGGG
117	CTAAATCCTTACGGGAATTTTGGCTGGTAC
118	CTACGTGCGCCGTTAGAACGGGTACACCCC
119	CTAGAGCCATGAGAATTGGCAATGTACGTG
120	CTATGATGTTACTCAGGGCTTTAACCCGAC
121	CTCACCGAAAGCTAGTTTAGGGGTGGGTAC
122	CTGAAGTAGCTAGTTGCCGACGGTAATCGA
123	CTGATGTGTTGGGGGTTGTAGTGCTGGGTT
124	CTGCGGGCGAGAGGTGCGCCAGCTTCTTCA
125	CTGGGGCAGACATCACTAGTAAACGTACTC
126	CTGGTTAGAGTATTCCAAGAGAGGGCTTAG
127	CTGTCGGTCTCGTCGCGTTTTTCTACCGGG
128	CTGTTAAGGAGGACTTTTCTTCAGATTCAA
129	CTTAAAAACGCCTTTTAAGCCTGGCTTTGA
130	CTTAGGAAGTGCATGAGTTGGTGCTGCCCT
131	CTTTCTGACCGCGGGTTCCTAAAATGTTTG
132	CTTTTTGAAGATAGGTTGTGTAACAGCCAA
133	GAATGGGGAGGACTAACTTGCAGGTGCCGT
134	GAATGGTTGATGTAACGTGCATTGTGCTAA
135	GACAATATGTGAAGTTTATACTCCAGTAGA
136	GACAGATTGCGCAATGTTGGGTATGGGTAC
137	GACCGGTGCAGATCCGTGCCATTGCGGTAC
138	GAGACAGATTGCGCAATGGGGGTTCTGGGG
139	GAGACAGATTGCGCAATGTTGAGCGGGGGC
140	GAGACTGTTTAACAATGCCCCCGAAAAAAC
141	GAGACTGTTTAGCATCTGTCTCTTAAAAAC
142	GAGCTTCCCGTGTTGATCACACGGCGGGCA
143	GAGGAAATAAGGGATGTATTGACACATGTG
144	GAGGACATTATCATGGTCAAAGTCGATTTT
145	GAGTCCTATCTTGGGGGGCCGAGGCAGGTG
146	GAGTGCGCAATGGGGATGGGGGGGCATTAC
147	GATACGATGTGGGGCAGGGTGGATATGTAC
148	GATACGATGTGGGGCAGGGTGGATATGTAC
149	GATAGTATTATTTTGTGTGTCAGTGGAAGT
150	GATGGTGTCTGTAAACTTCGATCAATGGTC
151	GATGTTGGTTCGAGTTTTGCCTGTAACGGA
152	GATTGCGCAATGAAGTCCAGGGGGTGGTCC
153	GATTGCGCAATGGGGCGGGTGGATATGTAC
154	GCAGATAAAGGATTATACCATATACATGAA
155	GCCCCTGCGCCTTAAGGGAGCCCAGAGCCG
156	GCGCAAATGATCCTACGCGTATGTTAAGGG
157	GCGCAATGACTCAGAAGGGGACGTAACGCG
158	GCGCAATGGGGGATACGGGTATGTTAAGGG
159	GCGCATTGTGGGGGTAGGGTGGATATGTAC
160	GCGGGCAGAAGTTGGATCCCGCTTACTGGA
161	GCGGTTGACGATTGATATACAGGCGCAGGC
162	GCGGTTGGAAGTACTGGGCGGGGGGCGAAT
163	GCGTTATTCGTGTCATGATATATTTAGGTA
164	GCTAACGTGGGGATTACAAGACGGGTTTAC
165	GGAATCCTGAAGCCGACGGTTTGCGTCCTT
166	GGACAAAGGATCGGCACAAAGGATCGGCAG
167	GGACATGGAGGGGTGGGCCTTATATATTGC
168	GGACTTCAGATCGCCGAGAGGGTGGAAGGA
169	GGAGACCCTCGTGTTCCGTGGCAGGTTCCC
170	GGAGAGCGAAACGATGGGAGGTGCGCCGTT
171	GGAGGGAGTAGGGAGTTGCGGGGCACGCCG
172	GGAGGTGTGATGGGCTTAAACGGTTGAGTG
173	GGAGGTTTTACGAAAGTGCGTAGGGGTGCG
174	GGAGTGCGAGGGGTTCACTAGAAATGCAGA
175	GGATAAATTTTCGCGTACATCCAAACATAG
176	GGATAACTTTAGTTGCACGCACTGTGTCTC
177	GGATACCACATTATGCTGAGATTGGCATCC
178	GGATCAATTTCTGTACTCGAGCTGAGTACG
179	GGCAGTTTCGCTGGTATATTGGGTGGAAAC
180	GGCCACCTTTGGGGGAGACATCTTTGGGTG
181	GGCGGGATGTTGGGGTAACAGCGGCAAGTC
182	GGCGTATTGACGGCGGAGACTAGAGTAACA
183	GGGAGTGCGTAGTACGTTGAACGGTGATAT
184	GGGCAATAGGTTTGGGGAGTAGTTAGGGTC
185	GGGCGGGTCGATCTCCACGTCGGGAGGATC
186	GGGGAAGAAGTTCTGACACTAACAGAGTAC
187	GGGTAGTGTCGGGGGTAGAGAGATTGGATC
188	GGGTGGCCGGGGAAGGACGTGGTGCACGTT
189	GGGTTAATTCTTTAGTGGTTGTTTCGCTAC
190	GGGTTTATTAACGGACCTATTTCTAATCGA
191	GGTCAAGTGGTAAACAAGTGCGCGTGGTAC
192	GGTCGCAAGGGCAGTGGTGGTCATAGCTAT
193	GGTCGGCTACAGGACCAGATGACAGTGTAC
194	GGTGGGAGAAGCACGGTCAGTGCGGTTGGA
195	GGTGTCAACGGACGGAATCGCAATTCATTG
196	GGTGTCTTTGACGAGTATTCTGATAACAGG
197	GTAAGGACTGATATGTCAGGCTTGGGACAC
198	GTATAGGAAAAGTGCCTAGGGAACGTTTTT
199	GTCAATAACAGGACTCTCGGAAAATCAGAC
200	GTCAGGACGCTGGACGCTGTTTGGTGTCGG
201	GTCGCGGGCCCTTTCAAAGTTAAAGTATCT
202	GTCTATATTCTCAGCGAGGGGTCGGCGATG
203	GTCTCATATTCGAGATTGGCGACGCAGGAC
204	GTCTGTGGGTGTGGCTGTGGTGGCGCACCA
205	GTGATGTAACAGTCGGAGTGTCCACGTGGT
206	GTGATTGAGGTCCTGTGGGTAGTGGTACCT
207	GTGATTGAGGTCTGTCTCTTATACACATCT
208	GTGCATCATGGTCCGCTTCTAGTTTGGCCC
209	GTGGAGGTTAATGGAAAGAACCTCGGGTAC
210	GTGGCATGACGGCGGTCAGCGCTATATGGT
211	GTGGTGGGGGCACCTGGTGAGAGGACGGTG
212	GTGGTGTTGAATGAATCTCTTGTGCATCTC
213	GTGGTTCAGTCTTGTGATATGAGTTGGATG
214	GTGTGTATGACCGGTTCTCTTCCGGAGTAC
215	GTGTGTGACG--------CGTGACGGGTAC
216	GTTATTCGGGATGGGTAGCATGGGGAGTGG
217	GTTCATGTGAGCAAGTTCTGTTTTTTTTTA
218	GTTCCAGAGGATTTTAGGATTTTATGCACT
219	GTTCCATGCGCTGTGATCCGGGGAGACGGG
220	GTTCCGAATGGTTTGGTAATTCTGAGCAGG
221	GTTCTACATCTAACATGTTGAAGTGATTGT
222	GTTCTCCTGTAATTACGCCAGGGAGTTGCC
223	GTTGAAAGGTGTAGGTGGGGGTTTCATGGG
224	GTTGAATGGGCAGTTGCGGGCAGGAACATG
225	GTTGCGACGACGGGGCGTGAAGGGGCCTTT
226	GTTGGCAAGTCGTAGATGTTCTAGGCACCG
227	GTTGGCGGTCCAGGGATATACCCTAGCGGC
228	GTTGGCTTTGACTGGAACTATTGACCGGGT
229	GTTGGCTTTTGGACGTAGCTCTATTAGTAC
230	GTTTAGGGTAAAATTCTAGGGTTGCGGTCA
231	GTTTTCGCTGACGGAGAGCGTCTGCATGTC
232	GTTTTGGGTTAATTAGAAGTAGCGTTTATT
233	TAAAAATGGTGTCGAGGAGCATAAATCGGA
234	TACATTCAGATAATCAGGTTAGAGCACTGT
235	TACGAGTCCCGTTGACATCGGGATTCAGAG
236	TACGTCGATATTTATCAAGCCTGGACACAC
237	TACTTGAGTGCAAGGGTCCGGTGTATTCTC
238	TAGAAGGATTGCGCACTAGTCTTTTAAGAC
239	TAGAGCTATGGTTAGGTTTGATGAATTTTC
240	TAGAGTGAATTATTGCACTTGCGGCCTCCC
241	TAGATGGTGTGGGAGGTACCCCATGTAGCG
242	TAGATGGTGTGGGAGGTACCCCATGTAGCG
243	TAGATGGTGTGGGAGGTACCCCATGTAGCG
244	TAGCACGTTGGGTAGAATAGGCAGGGCGAC
245	TAGCGGTGTGAAAGATACCGGGTTATCTGA
246	TAGTAGCATGAAGGGAAGGAGGAAAACTTG
247	TAGTATGGACTGTTGACCGAACTAATAGTT
248	TATGGGGGTTCAAGGGGAGCATGCACGTAC
249	TATGTATATTCGTGTTAGTCGGACCCTCCG
250	TATTATTTCGGCCTTTCAGTGACATTTAGT
251	TATTGCTAAAAGAATGTGTGTTCGGTTGTA
252	TCACGGGCAGTAGAAGAGCTGTGGGGTAAC
253	TCAGTACTCGTTCAGTAGTGGTGCTGCAAT
254	TCCATTGGACTTATTATTCGTTATAGGTAC
255	TCCCCGCCGGAACGAAACCCCAAACCGGAC
256	TCCCTGGCAGTTTTGAAGCGAAATGCAAGA
257	TCCGGCGTGGTCTAGGGCGCGAATCGCATG
258	TCGAATTGATCTTGCGTTAATGGGTAGTAC
259	TCGAGATGGCTATGTAAATGGTGATGCCGC
260	TCGAGGAGCGGCGTTGAGGGGTTTCTGGGG
261	TCGATCTACGATATATGGCGAGTTTCAAAC
262	TCGCTATTCATCGCGCGGTTCAGCGTGAAC
263	TCGGGTCGAGTCTCCGGTCGTCGTGTACAG
264	TCGGTACCTCAGTGGACCGTTAATATTGAC
265	TCGTGCGGCTATAAGTCCAGAGGGTGGTCC
266	TCTAGGGGTTTGGCCAAGGTGGAGAAAAGC
267	TCTAGGGGTTTGGCCAAGGTGGAGAAAAGC
268	TCTCACCGAAAGCTAGTTTAGGGGTGGGTA
269	TCTCTAAAGTATGTCGGAGGCGGGCGGCGC
270	TCTTAACTCAATGGCACTGCGATCGGTGGG
271	TCTTAACTTTAGGAGCTCCGAAGACGCATG
272	TCTTCATAACGAGCTTAGTACACTGGAACA
273	TCTTCGCTAATCACTCTAAATCTCGCTCTT
274	TGAGACGAGAAAAAGTCCACAAGGGGGAAC
275	TGAGACGAGAAACCGTGGACGATTGGGAAG
276	TGAGAGGACATCAACCTAGTGCGGGCATTG
277	TGAGCAATAAGGGTGATACGCGACAGTTAA
278	TGAGGCTGCAGGAATTATCCTCCGTCTTGC
279	TGATGGGGAGGGGGGGGTCGTCGGAAGTAC
280	TGATGTATAAACCCTCTTTAAGAC------
281	TGCAGACTTCATTGGGCGGGGCGGAGGTAC
282	TGCGAGCGCAGAGTTATAAGACGTAACGCG
283	TGCGCAATGTCGATGGGGGGGTATGGGTAC
284	TGCTGGGGCTTTATTGCGGGTTGACTGGCA
285	TGCTGTGGGGTGTGTGACCCCCGCTAGGTC
286	TGCTTGTAGTTGGTAGGCCCGGCGGGCTTG
287	TGGAATTTACCCTAAGGAAAGAGTCTGAGA
288	TGGACTAGGAGGCTTGAGCCCGGGTATCCC
289	TGGACTAGGAGGCTTGAGCCCGGGTATCCC
290	TGGAGGTTAATGGGAAGTACTCTCGGGTAC
291	TGGAGGTTAATGGGAAGTACTCTCGGGTAC
292	TGGCACCGTGGTACAGTAGCGAGGATTATC
293	TGGGCGGTTGGGTTGTCTTGCGAAATTTTG
294	TGGGGAGGGGGGGGGTCATGCGCGGCATAC
295	TGGTGAGAGGCGGGCATGAATACAGGGGTG
296	TGTACAGGGCCCTTGGTGCGGTATCCAACA
297	TGTAGGTGCGAGTTGTTGCGGATGTGCTCG
298	TGTATCAAGCCTTGAATAAAACGCCTAACG
299	TGTATTTAAGACGCGGTATATTATGGTTAG
300	TGTCAATAACAGGACTCTCGGAAATCAGAC
301	TGTGATCCCGGGTTAAATGCTGTAACGGGC
302	TGTGGGAGTGAGGTGAAGTCAGACCGTGTC
303	TGTTCCTTTGGGTCAGTTGACCGGGACGTG
304	TTACAAGTTAGGCCTAGGTACATGACGCGC
305	TTACTGGTGTATCTGGATCCCGAGGAAAGG
306	TTAGGGGCTGCGTGGTAGAAGAATCCTCCG
307	TTAGGGGCTGCGTGGTAGAAGAATCCTCCG
308	TTAGTGTATTCTAGACTCAATGACGGCGAC
309	TTCATCCTGTGGCACAGTTTGGAATGGTGC
310	TTCATGTGAGCAAGTTCTGTTTTTTTTTTA
311	TTCGTGTGGATGCGCACGCTAGGTTTCGTG
312	TTCTAGCAGAAGAGTATATTGCCTGTCTCT
313	TTCTAGCAGAAGAGTATATTGCGTAGGGGA
314	TTCTCCGTTGGGTTGGTGCCTCATGCTTCA
315	TTCTGAGCGTCGGTCAGCGTAGTGGTCTAG
316	TTGCGAACTGGGGGCAGGGGGGATATGTAC
317	TTGCGCAATGTGGGGCAGGGGGATATGTAC
318	TTGGAAGCGTTGCTGAAAGGCAAAAGATTC
319	TTGGACTTATTGTATTTGCAGCAGCGGGGA
320	TTGGGCATCCGTTGCGCACAATCTTGGTAC
321	TTGGGGAGTTGGTTCGGATTTTTTTAGGAA
322	TTTAAGCTCGGGTGTGGCGGGGAGTGTCCC
323	TTTATTGGTTCGCTGCTGAGATAGCGTACC
324	TTTCAAATTGGTTGGAAGTGTACACTGGCA
325	TTTCAAATTGGTTGGAAGTGTACACTGGCA
326	TTTCTTTACAGACCGGGGTGCCGTGACGGG
327	TTTGAGGGGTGTCGCGAAATACCGGGGGGA
328	TTTGGCCTTGACCTCAGTGCTTACAATTGA
329	TTTTAAGTTAATTCTCGTCGCTCTTTTAGG
330	TTTTTATTCACCAATTGGGATTGACGGAGC
331	TTTTTGCATCTAGCACACAGGGCGGTTGCC
