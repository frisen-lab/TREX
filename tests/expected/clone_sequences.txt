clone#	clone_id
1	ACTAGGAGATTGACGGATCACCTTTGGTCG
2	GGCGGAGACGGCACACACCGAGGGTCAGTT
3	GGTCTCCCTATACCAACAGTATCGTCTCAA
4	TGTCAATCGTTCGGTTGAGCAAGATCTTAG
